scripts/EN/Harry

```diff
@@ -3302,9 +3302,9 @@ Area(0x5b); // 0x3fc4: 00001026
 CleanScreen(); // 0x3fca: 00000064
 LoadBackground(0x53); // 0x3fce: 00001069
 DrawBackground(); // 0x3fd4: 00000065
-ShowText((1287)"I move past one of the sinks and notice an empty pet bottle sitting on the end of the counter."); // 0x3fd8: 0000100c
+ShowText((1287)"I move past one of the sinks and notice an empty plastic bottle sitting on the end of the counter."); // 0x3fd8: 0000100c
 HandleInput(); // 0x3fde: 00000010
-ShowText((1288)"¥"Perhaps this might come in handy for something.¥" I mutter this and grab the empty pet bottle. Then I leave the kitchen."); // 0x3fe2: 0000100c
+ShowText((1288)"¥"Perhaps this might come in handy for something.¥"¥nW2I mutter this and grab the empty plastic bottle. Then I leave the kitchen."); // 0x3fe2: 0000100c
```

Comment:
Changed pet to plastic to match the PS1 version. (PET refers to the type of plastic and is how they refer to these bottles in Japan.)